README.md

```diff
@@ -1,180 +1,108 @@
 # RFC Process
 
-This document describes the RFC process for the [npm
-CLI](https://github.com/npm/cli), and provides a way for the [npm CLI
-team](https://github.com/orgs/npm/teams/cli-team) and the wider community
-to have discussions about the features and direction of the package
-manager.
+This document describes the RFC process for the [npm CLI](https://github.com/npm/cli), and provides a way for the [npm CLI team](https://github.com/orgs/npm/teams/cli-team) and the wider community to have discussions about the features and direction of the package manager.
 
 ## What is an RFC?
 
-The name is a reference to the **IETF's Request For Comments** process, and
-involves a document or series of documents which are drafted, reviewed, and
-eventually ratified (approved) by the npm team through discussion
-among those interested, both within and outside of the npm team.
+The name is a reference to the **IETF's Request For Comments** process. In the case of the npm CLI these are implemented in the form of a GitHub issue in this repository. The issue will be drafted, reviewed, discussed and eventually ratified by the npm team with the help of our community members during our RFC review calls.
 
-An RFC can propose any change to the npm CLI itself, and may include npm
-registry changes meant to support that CLI change.
+An RFC can propose any change to the npm CLI itself, and may include npm registry changes meant to support that CLI change. It is also important to be aware that an RFC being ratified does not guarantee that it will be implemented. The npm CLI team welcomes outside contributors, and these approved RFCs are a good way to find tasks to work on.
 
 ## Rough Consensus
 
-The development of new features within the npm CLI follows a [Rough
-Consensus](https://tools.ietf.org/html/rfc7282) model, similar to the IETF.
+The development of new features within the npm CLI follows a [Rough Consensus](https://tools.ietf.org/html/rfc7282) model, similar to the IETF.
 
-The following points are intended to help you understand and participate in
-this process productively.
+This consensus is gathered from the npm CLI team, as well as all community attendees of RFC review calls as well as community members who choose to comment on an RFC issue. If a community member has an objection, it is appreciated if they are also able to attend an RFC review call in order to have a real time conversation about their concerns.
```

Comment:
It mentions that consensus is gathered but no real call out to how objections are dealt with

Reply:
isn't that what "rough consensus" above covers? (not that we've really ever had minority objections to things in RFC calls)